src/utils/api.js

```diff
@@ -41,11 +41,11 @@ export const handleLogin = async (user) => {
       // read more in mdn: https://developer.mozilla.org/en-US/docs/Web/API/Response/json#examples.
       // We expect the body returned by the backend to look like {"user":{"name":"akos","userId":"649496075cb6d83f666e9f3a","role":"user"}}
       const body = await response.json()
-      // return just the user object to the caller of `handleLogin` (which is being called in `Login.js` component)
-      return body.user
+      // return an object with the success state and the user object to Login.js
+      return { successful: true, user: body.user }
     } else {
       console.log('Login failed')
-      return false
+      return { successful: false }
```

Comment:
Here I am  making sure that the return value of `api.js` matches what's expected/used in the Login.js file when it destructures this response.